### PR TITLE
Add Double MNIST dataset

### PR DESCRIPTION
**Pull request by tesfaldet**

### CODE REVIEW

**Comment by tristandeleu** [comment]
This is truly amazing, thank you very much for this new dataset!
Everything looks good to me, just a tiny comment on the choice of default values, but I am going to merge it for now since this can be fixed later on.

File: torchmeta/datasets/helpers.py [hunk 1]
```diff
@@ -279,3 +279,59 @@ def cub(folder, shots, ways, shuffle=True, test_shots=None,
...
+        warnings.warn('Both arguments `ways` and `num_classes_per_task` were '
+            'set in the helper function for the number of classes per task. '
+            'Ignoring the argument `ways`.', stacklevel=2)
+        ways = kwargs['num_classes_per_task']
+    if 'transform' not in kwargs:
+        kwargs['transform'] = Compose([ToTensor()])
+    if 'target_transform' not in kwargs:
+        kwargs['target_transform'] = Categorical(ways)
+    if 'class_augmentations' not in kwargs:
+        kwargs['class_augmentations'] = [Rotation([90, 180, 270])]
```
Comment (tristandeleu) [inline torchmeta/datasets/helpers.py]: Do you think that adding rotated variants of the images is a good default you had in mind for Double-MNIST @shaohua0116 (similar to how that's done for Omniglot)?

Comment (tesfaldet) [reply torchmeta/datasets/helpers.py]: Hmmm, come to think of it, I'm not sure what the best default would be! Perhaps I should leave it empty?

Comment (tristandeleu) [reply torchmeta/datasets/helpers.py]: I modified it and left it empty for now (`class_augmentations=None`), waiting for Shao-Hua's input.

**Pull request merged**